packages/server/src/drizzle/videos/videos.dto.ts

```
@@ -0,0 +1,33 @@
+import { DBConnection } from "../../drizzle/drizzle-client";
+import { PaginationSchema } from "../../utils/pagination";
+import { safeAwait } from "../../utils/safeAwait";
+import { CreateVideoSchema, videosTable } from "./videos";
+import { count } from "drizzle-orm";
+
+export class VideoTable extends DBConnection {
+	constructor(TURSO_CONNECTION_URL: string, TURSO_AUTH_TOKEN: string) {
+		super(TURSO_CONNECTION_URL, TURSO_AUTH_TOKEN);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)